R/RLearner_regr_randomForest.R

```diff
@@ -66,6 +68,18 @@ trainLearner.regr.randomForest = function(.learner, .task, .subset, .weights = N
   } else {
     data = getTaskData(.task, .subset, target.extra = TRUE)
     m = randomForest::randomForest(x = data[["data"]], y = data[["target"]], ...)
+    ## ugly hack
+    ## if the formula interface is used the training data is saved, but not with the default interface
+    ## the training data is used when predict.type = "se" and calibration is not FALSE
+    if (.learner$predict.type == "se") {
+      if (is.null(.learner$par.vals$calibrate)) {
+        m$data = data[["data"]]
+      } else {
+        if (.learner$par.vals$calibrate) {
+          m$data = data[["data"]]
```

Comment:
most of this pr is adding this functionality, which we were using in that other project (so sensible it was in that branch). i'd suggest a new pr or a rebase to just include the minor changes. also why is this high priority?